store/actions/crypto.actions.tsx

```diff
@@ -21,3 +23,28 @@ export const readData = () => {
     }
 }
 
+export const addCripto = (textInput: string) => {
+    return async (dispatch: Dispatch<Action>, getState: any) => {
+        const { cryptoList } = getState().cripto
+
+        const newCoin: Crypto = cryptos.find((crypto: Crypto) => {
+            return (textInput === crypto.name || textInput === crypto.initials)
+        })
+
+        const addedCrypto = cryptoList.findIndex((crypto: Crypto) => crypto.id === newCoin?.id)
+
+        if (addedCrypto === -1 && newCoin) {
```

Comment:
[minor] You might as well do a single search in newCrypto by implementing a nested find or findIndex; it is better to encapsulate this behavior on a single variable rather than using two